src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -1118,6 +1118,13 @@ def forward(
         hidden_states = self.layernorm_embedding(hidden_states + embed_pos)
         hidden_states = nn.functional.dropout(hidden_states, p=self.dropout, training=self.training)
 
+        if self.gradient_checkpointing and self.training:
+                if use_cache:
+                    logger.warning_once(
+                        "`use_cache=True` is incompatible with gradient checkpointing. Setting `use_cache=False`..."
+                    )
+                    use_cache = False
```

Comment:
```suggestion
            if use_cache:
                logger.warning_once(
                    "`use_cache=True` is incompatible with gradient checkpointing. Setting `use_cache=False`..."
                )
                use_cache = False
```

Reply:
OK, just fixed indentation with another commit. Thanks!

Reply:
Sure! For some reason, I cannot seem to install this tool properly. Are there any guides on how to install this for the Transformers repository?

Reply:
if you are inside `transformers` repo, you should be able to just run `make fixup` and it should work. What are the errors you get?

Reply:
I use VSCode, and this is the output in my console when I try to run `make fixup` on the file: 

```
pmollerus@Philips-MacBook-Pro transformers % make fixup src/transformers/models/time_series_transformer/modeling_time_series_transformer.py
make: python: No such file or directory
No library .py files were modified
python utils/custom_init_isort.py
make: python: No such file or directory
make: *** [Makefile:60: extra_style_checks] Error 127
```

I used brew to install make for this project.

Reply:
I see, can you try to run `make fixup` without `src/transformers/models/time_series_transformer/modeling_time_series_transformer.py` ?

Reply:
When I run that, I get this output:

```
pmollerus@Philips-MacBook-Pro time_series_transformer % make fixup
make: *** No rule to make target `fixup'.  Stop.
```

I also tried running `make fixup modeling_time_series_transformers.py` in the time series transformer directory and receive the same output.

Reply:
I see, can you set yourself on the root of `transformers` repo? i.e. where the [`Makefile` lives](https://github.com/huggingface/transformers/blob/main/Makefile), that should fix the issue I believe

Reply:
Doing this, I receive the following output: 

```
pmollerus@Philips-MacBook-Pro transformers % make fixup
make: python: Command not found
No library .py files were modified
python utils/custom_init_isort.py
make: python: No such file or directory
```

Reply:
Ahh this is because python is not installed on your environment I guess, I can run the command for you if you want, just let me know